packages/discord.js/typings/index.d.ts

```diff
@@ -6542,6 +6558,23 @@ export interface PartialMessage
 
 export interface PartialMessageReaction extends Partialize<MessageReaction, 'count'> {}
 
+export interface PartialPoll
+  extends Partialize<
+    Poll,
+    'allowMultiselect' | 'layoutType' | 'expiresTimestamp',
+    null,
+    'question' | 'message' | 'answers'
+  > {
+  question: { text: null };
+  message: PartialMessage;
+  // eslint-disable-next-line no-restricted-syntax
```

Comment:
Not a blocker but a question, what does this comment disable, I see no restricted syntax 😅 👀

Reply:
Because of the rule added in this pull request, but the rule shouldn't apply to this interface as it's not user input
- #10045